docs/ref/templates/builtins.txt

```diff
@@ -38,6 +29,42 @@ Sample usage:
         {{ body }}
     {% endautoescape %}
 
+When auto-escaping is in effect, all content derived from variables have HTML
+escaping applied to it before placing the result into the output (but after any
+filters have been applied). This is equivalent to manually applying the
+:tfilter:`escape` filter to each variable.
```

Comment:
Maybe:

```suggestion
When auto-escaping is in effect, all content derived from variables has HTML
escaping applied before placing the result into the output (but after any
filters are applied). This is equivalent to manually applying the
:tfilter:`escape` filter to each variable.
```

Reply:
Thanks! Applied (originally I did not touch this paragraph since I just moved it to after the example).